example/Makefile

```diff
@@ -1,26 +1,32 @@
-.PHONY: default run build clean
-
-SRCS := $(wildcard ../*.go)
+SHELL := bash
+.SHELLFLAGS := -eu -o pipefail -c
+.DELETE_ON_ERROR:
 
+.PHONY: default
 default: run
 
-x.wasm: main.go $(SRCS)
+srcs := $(wildcard ../**/*.go)
```

Comment:
styleguide suggested that all variables that are not Makefile specific vars or env variables should be lowercase and positioned nearest to the point of use